hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -212,6 +212,32 @@ def _bank_add_signals(
     }
 
 
+@bp.route("/bank/<bank_name>/content/<content_id>", methods=["PUT"])
+def bank_update_content(bank_name: str, content_id: int):
+    """
+    Update the metadata for a banked content item.
```

Comment:
blocking: Can you document the parameters accepted?

Reply:
should we allow updating anything besides `disable_until_ts`? I could see `collab_metadata` or `original_media_uri` as candidates but not sure what they do